tests/crashes/README.md

```diff
@@ -14,3 +14,10 @@ If you happen to fix one of the crashes, please move it to a fitting
 subdirectory in `tests/ui` and give it a meaningful name.
 Also please add a doc comment at the top of the file explaining why
 this test exists. :)
+Adding
+Fixes #12456
+Fixes #98765
+to the commit message or title of your pull request will ensure the
```

Comment:
```suggestion
to the commit message of your pull request will ensure the
```

Putting it in the title is considered bad style cuz it's not clickable, not descriptive and may lead to ppl adding a proper title.

Furthermore, I don't think GH recognizes those in the title. https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#about-linked-issues-and-pull-requests only mentions the PR description.

Reply:
Ah, and putting it in the commit message is also not great because GH continuously “mentions” the issue on rebases.

Reply:
Ah yes sorry, I mean this  "body"/"header" or whatever this is called? (the first comment contains the commit message by default)